### ISSUE TITLE
 # Evaluate model

### ISSUE THREAD

**Issue by lyuwenyu** [aymericdamien/TensorFlow-Examples#127]
```
# Construct model
pred = conv_net(x, weights, biases, keep_prob)

# Define loss and optimizer
cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=pred, labels=y))
optimizer = tf.train.AdamOptimizer(learning_rate=learning_rate).minimize(cost)

# Evaluate model
correct_pred = tf.equal(tf.argmax(pred, 1), tf.argmax(y, 1))
accuracy = tf.reduce_mean(tf.cast(correct_pred, tf.float32))
```

Here variable *pred* is *logit*,  use it to compute *correct_pred* may be meaningless, so I think computing *correct_pred*  should modified as below :  
```
correct_pred = tf.equal(tf.argmax( tf.nn.softmax(pred), 1), tf.argmax(y, 1))
```

**Comment by sunnysai12345** [comment]
For evaluating model we are taking the index of the max value obtained and is compared to target, which is similar whether we use softmax or not, applying Softmax just help us reducing these values to probabilities which are not required in this case.
